src/query.ts

```diff
@@ -74,6 +88,38 @@ export class QueryBuilder<D extends Document> {
     return this;
   }
 
+  // Returns an object used to build a clause that filters documents that match
+  // any of the clauses in the group.
+  public getOneOfFilterGroupBuilder(): OneOfFilterGroupBuilder {
+    return new OneOfFilterGroupBuilder();
+  }
+
+  public withOneOfFilter(
+    options: { field: string; terms: (string | null)[] }[]
+  ): QueryBuilder<D> {
+    const group: Query[] = [];
+    options.forEach(({ field, terms }) => {
+      if (terms.includes(null)) {
+        group.push({ bool: { must_not: { exists: { field } } } });
+      }
+
+      const values = terms.filter((s): s is string => !!s).map((s) => `${s}`);
```

Comment:
Acho que aqui tu poderia utilizar uma função como predicado, por exemplo: `terms.filter(this.termsWithValue).map(this.transformToString);`

Reply:
Feito aqui 13f04712937ec @mahenrique94, valeu pela sugestão!